### ISSUE TITLE
Client - index range error when entering blank input

### ISSUE THREAD

**Issue by RC-13** [hummingbot/hummingbot#3737]
**Describe the bug**
Opening ticket for tracking

* When entering a blank input in the Hummingbot client, it triggers the following error:

```
2021-07-15 23:38:41,093 - 1 - hummingbot.client.hummingbot_application - ERROR - list index out of range
Traceback (most recent call last):
  File "/home/hummingbot/hummingbot/client/hummingbot_application.py", line 142, in _handle_command
    if command_split[0] == s['command']:
IndexError: list index out of range
```

although it doesn't cause any issue. This behavior can occur when using a docker build and source build.

**Steps To Reproduce**
Here are the steps to reproduce the issue (see attachments in the section below):

1. Open the Hummingbot client 
1. Press enter multiple times without entering any input from the input pane.
...

**Screenshots**
![image](https://user-images.githubusercontent.com/57189990/125872068-03bbc478-1303-412a-9d93-53d78f66a889.png)

**Release version**
0.41.0

**Attachments**
// Please attach your config file and log file to help us debug. It would be difficult for us to help you without those! 

_WARNING: Do NOT publish any exchange API keys or your wallet's private key. Whoever has access to them may steal your assets!_

**Optional: your discord username:**

┆Issue is synchronized with this [Clickup task](https://app.clickup.com/t/1ramt6d) by [Unito](https://www.unito.io)

**Comment by EmmeCorelli** [comment]
**Hello, it's me**

Change the line 140 in client/hummingbot_application.py from

                if shortcuts is not None:

in

                if shortcuts is not None and len(command_split):

m.c.

**Comment by rapcmia** [comment]
Fixed on staging (dev-1.2) and development branch
